packages/babel-plugin/package.json

```diff
@@ -2,15 +2,15 @@
   "name": "@compiled/babel-plugin",
   "version": "0.6.14",
   "description": "A familiar and performant compile time CSS-in-JS library for React.",
-  "author": "Michael Dougall",
-  "license": "Apache-2.0",
   "homepage": "https://compiledcssinjs.com",
   "bugs": "https://github.com/atlassian-labs/compiled/issues/new?assignees=&labels=bug&template=bug_report.md",
   "repository": {
     "type": "git",
     "url": "https://github.com/atlassian-labs/compiled.git",
     "directory": "packages/babel-plugin"
   },
+  "license": "Apache-2.0",
```

Comment:
Hey @MonicaOlejniczak , out of curiosity, what's the reason to place these fields below?

Reply:
I used [sort-package-json](https://github.com/keithamus/sort-package-json) which defines the sort order, and it is loosely based off the order specified in https://docs.npmjs.com/cli/v7/configuring-npm/package-json